Automattic/jetpack
# Refactor regeneration notice

This patch refactors how the Critical CSS regeneration notice is displayed.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Apply the patch, change the theme, make sure the regeneration notice appears and dismiss it.
